src/Uno.UI/UI/Xaml/Window.skia.cs

```diff
@@ -50,11 +50,10 @@ internal void OnNativeSizeChanged(Size size)
 			}
 
 			Bounds = newBounds;
+			ApplicationView.GetForCurrentView().SetVisibleBounds(newBounds);
```

Comment:
@MartinZikmund FYI as this will create a merge conflict with multiwindow PR.

We need to ensure that visible bounds is set properly before we raise size changed because subscribers can read this value.

Reply:
Any chance this change can be test covered? Maybe via some test hook?

Reply:
@MartinZikmund I wasn't able to write a test, but added in assert in SamplesApp. Not sure how the assert will behave on Android though (maybe the sizes I'm comparing could have differences due to status bar)

Reply:
In Wasm implementation, there is this comment:

https://github.com/unoplatform/uno/blob/72c0786f059068144b7f5b0c2f4dac65bdd68b86/src/Uno.UI/UI/Xaml/Window.wasm.cs#L117-L118

So it seems the change is actually wrong, which means we shouldn't rely on the right visible bounds when we are raising size changed.

Reply:
In WinUI, `ApplicationView` is unusable. It always throws an exception.
In UWP, SizeChanged is raised before VisibleBoundsChanged, however, the new VisibleBounds value can be correctly read from SizeChanged event.

So, the sequence is like:

- Window.Bounds and ApplicationView.VisibleBounds are both updated.
- Window.SizeChanged is raised
- ApplicationView.VisibleBoundsChanged is raised

Currently, we raise VisibleBoundsChanged as soon as we update the visible bounds. This is not correct. Since the fix will involve more code that has been changed in multiwindow, I'll wait for this PR.